src/shabdawali.js

```diff
@@ -231,9 +231,24 @@ Shabdawali.prototype.typeText = function(cLine){
                 //this.element.textContent  += char;
                 this.element.textContent  = cLine.substr( 0, this.currentLetterIndex  );
                 var that = this;
-                setTimeout(function() {
+                if(char == ' ')
+                {
+                    var prevSpaceIndex = cLine.lastIndexOf(' ', this.currentLetterIndex - 2);
+                    if(prevSpaceIndex == -1)
+                    {
+                        prevSpaceIndex = 0;
+                    }
+                    var prevWord = cLine.substr(prevSpaceIndex, this.currentLetterIndex- prevSpaceIndex);
+                    setTimeout(function() {
+                        that.typeText(cLine);
+                        }, 50 * prevWord.length );
```

Comment:
I believe it should be related to the speed set by user. Eg this.speed + 15 * prevWord.length . Moreover, as we want to give long pause only in case of complex words, you may create a function to check complexity of a word. Eg 
length > 6, 
number of consonants > length/2 || number of consonants > 4 
word ends with full stop, exclamatory sign or semi colon

Reply:
Okay, so no pauses for less complex words, and pauses only for words that satisfy the above condition.

Reply:
Yes, in my opinion. Otherwise it'll slow down typing and user'll not be able to control it. Moreover, I believe there should be a flag/options "dynamicPause". If the value of "dynamicPause" is true then only this change should be applied.

Reply:
Okay I haven't added the flag but I'll do so tomorrow. Let me know if anymore changes are required to the updated commit.